lib/connect_proto/src/order.proto

```diff
@@ -98,6 +99,7 @@ message Order {
     ResultStatus results_status = 18; // Current overall status of the order
     ResponseFlag response_flag = 19; // Specificity of the response requested from the receiving system
     repeated Identifier external_ids = 20; // May contain any number of ids related to this order
+    repeated Results.Result results = 21; // If this order already has a result, add the result here
```

Comment:
I find this slightly confusing... Should `Result` just be a top level proto rather than nested under `Results`?

Reply:
and/or
```suggestion
    Results results = 21; // If this order already has a result, add the result here
```